wallets/cosmostationextention.json

```diff
@@ -0,0 +1,71 @@
+{
+  "$schema": "../wallet.schema.json",
+  "wallet_name": "cosmostationextention",
+  "pretty_name": "cosmostation",
+  "website": "https://www.cosmostation.io/",
+  "git_repo": "https://github.com/cosmostation/cosmostation-chrome-extension",
+  "supported_chains": [
+    "cosmoshub",
+    "akash",
+    "assetmantle",
+    "axelar",
+    "bandchain",
+    "bitcanna",
+    "bitsong",
+    "cerberus",
+    "chihauha",
+    "comdex",
+    "crescent",
+    "cryptoorgchain",
+    "cudos",
+    "desmos",
+    "emoney",
+    "evmos",
+    "fetchhub",
+    "gravitybridge",
+    "injective",
+    "iris",
+    "impacthub",
+    "juno",
+    "kava",
+    "ki",
+    "konstallation",
+    "kujira",
+    "likecoin",
+    "lumnetwork",
+    "panacea",
+    "nyx",
+    "omniflix",
+    "onomy",
+    "osmosis",
+    "passage",
+    "persistence",
+    "provenance",
+    "regen",
+    "rizon",
+    "secretnetwork",
+    "sentinel",
+    "shentu",
+    "sifchain",
+    "sommelier",
+    "stafihub",
+    "stargaze",
+    "starname",
+    "stride",
+    "teritori",
+    "tgrade",
+    "umee",
+    "xpla"
+  ],
+  "features": [
+    "suggest_chain"
```

Comment:
Do you have documentation on this feature? (Will help our further design of this feature)

Reply:
Found it! https://docs.cosmostation.io/docs/Developer%20Guide/Cosmostation%20Extension/chains/#add-chains